vms/platformvm/client.go

```diff
@@ -119,7 +121,7 @@ type Client interface {
 	// GetTimestamp returns the current chain timestamp
 	GetTimestamp(ctx context.Context, options ...rpc.Option) (time.Time, error)
 	// GetValidatorsAt returns the weights of the validator set of a provided
-	// subnet at the specified height.
+	// subnet at the specified height or at proposerVM height if [isProposed] is true.
```

Comment:
I think this comment needs to be updated now.

Also - thoughts on changing `height uint64` to `height api.Height`? It feels like right now the `MaxUint64` behavior might feel odd

Reply:
I made the change since I don't feel strongly one way or another. I agree that the behavior might be unintuitive but accepting `height uint64` had the benefit of not changing the interface at all.